src/forking-store.js

```diff
@@ -11,7 +11,9 @@ import {
 const BASE_GRAPH_STRING = "http://mu.semte.ch/libraries/rdf-store";
 
 export default class ForkingStore {
-  #internalStore = new Store();
+  // See https://github.com/redpencilio/forking-store/issues/32 when considering using
+  // the `Store` constructor instead of `graph()`
+  #internalStore = graph(); // Is not actually a graph but rather a store, bad naming from rdflib
```

Comment:
You could also alias the import name instead of the comment, but this is fine as well :+1: